specification/labservices/resource-manager/Microsoft.LabServices/stable/2022-08-01/LabServices.json

```diff
@@ -0,0 +1,301 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-08-01",
+    "title": "LabServicesClient",
+    "description": "Azure Lab Services REST API.",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.LabServices/operations": {
+      "parameters": [
```

Comment:
The parameters should be in "get", not path level.

Reply:
Mmm is that really the issue? Because that's what we had in the last preview version we released https://github.com/Azure/azure-rest-api-specs/blob/80065490402157d0df0dd37ab347c651b22eb576/specification/labservices/resource-manager/Microsoft.LabServices/preview/2021-11-15-preview/LabServices.json#L43

Reply:
Had a separate discussion on this. This should be resolved once we deploy our API in production.